README.md

```diff
@@ -24,8 +24,9 @@ Execute `help(pytaxonkit.name2taxid)` (and so on) from the Python interpreter fo
 
 > **NOTES**
 > - The `reformat` operation is automatically run by `pytaxonkit.lineage` and cannot be invoked independently.
-> - The `taxid-changelog` operation is not supported.
 > - The `genautocomplete` operation is specific to the shell and is not supported.
+> - Several other operations are not supported, including `cami-filter`, `create-taxdump`, `profile2cami`, and `taxid-changelog`.
+> - The `pytaxonkit.__version__` variable refers to the version number of the Python bindings, while the `pytaxonkit.__taxonkitversion__` variable corresponds to the version of the installed TaxonKit program. These version numbers are not necessarily equal.
```

Comment:
Some updates to the README, noting additional TaxonKit features not supported in the Python bindings.